tests/results/DiffTest.py

```diff
@@ -17,6 +17,10 @@ def test_add_lines(self):
         self.uut.add_lines(0, ['t'])
         self.uut.add_lines(0, [])
 
+    def test_add_line(self):
+        self.uut.add_line(0, 't')
+        self.assertRaises(ConflictError, self.uut.add_line, 0, 't')
```

Comment:
could you make a test whether successfully a line was added? Meaning to check the diff stats / changed version.